llvm/include/llvm/Transforms/SandboxIR/SandboxIRValues.def

```diff
@@ -0,0 +1,29 @@
+//===- SandboxIRValues.def --------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// clang-format off
```

Comment:
is this really necessary? no other `.def` file in `llvm/include` has `clang-format off`

Reply:
At this point it is not necessary, but later on there are some entries that look like this:
```
DEF_INSTR(BinOp, OPCODES(\
                                                    OP(Add)   \
                                                    OP(FAdd) \
                                                    OP(Sub)   \
                                                    OP(FSub) \
                                                    OP(Mul)    \
                                                    OP(FMul)  \
```
and clang-format won't list the OP() entries vertically to save vertical space. Anyway, I will remove it for now.